CSS/style.css

```diff
@@ -0,0 +1,184 @@
+@import url('https://fonts.googleapis.com/css2?family=Clicker+Script&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,900;1,600;1,700;1,800;1,900&display=swap');
+
+body {
+    margin: 0%;
+    padding: 0%;
+    background-color: #FFF;
+}
+
+.My_logo {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 22px;
+    line-height: 20px;
+    color: #6070ff;
+}
+
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    padding: 26px 164px 26px 155px;
+    align-items: center;
+    padding-bottom: 4%;
+    background-color: white;
+}
+
+.Menu_icon1 {
+    width: 30px;
+    height: 3px;
+    background-color: #6070ff;
+    margin: 8px 0;
+}
+
+.body_section {
+    font-family: 'Poppins', sans-serif;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    border-bottom-left-radius: 70px;
+    background-color: #FFF;
+}
+
+.body_section_title {
+    width: 80%;
+    margin: auto;
+    padding-top: 10%;
+    padding-bottom: 20%;
+}
```

Comment:
- [ ] While you did a wonderful job implementing the required changes suggested by the previous reviewer :+1:, as mentioned before as a project requirement your design should match as much as possible with the Figma template. Kindly, justify your items to take the whole width of the nav bar and check the required background for the main section as you can see below :100: 

**Your design**

![Screenshot from 2023-02-17 07-40-32](https://user-images.githubusercontent.com/105117832/219657087-2337145b-0278-485f-8c3c-bb413311f813.png)

![Screenshot from 2023-02-17 07-40-58](https://user-images.githubusercontent.com/105117832/219657118-3db5ee1a-f300-4aa0-bd53-55cef71c2115.png)

**Figma**

![Screenshot from 2023-02-17 07-41-54](https://user-images.githubusercontent.com/105117832/219657182-0c7ca2e0-0863-4622-a0d8-349fce48c975.png)

![Screenshot from 2023-02-17 07-42-06](https://user-images.githubusercontent.com/105117832/219657193-88c02f51-12f5-4396-bef6-400a10f9a446.png)

Reply:
> * [ ]  While you did a wonderful job implementing the required changes suggested by the previous reviewer 👍, as mentioned before as a project requirement your design should match as much as possible with the Figma template. Kindly, justify your items to take the whole width of the nav bar and check the required background for the main section as you can see below 💯
> 
> **Your design**
> 
> ![Screenshot from 2023-02-17 07-40-32](https://user-images.githubusercontent.com/105117832/219657087-2337145b-0278-485f-8c3c-bb413311f813.png)
> 
> ![Screenshot from 2023-02-17 07-40-58](https://user-images.githubusercontent.com/105117832/219657118-3db5ee1a-f300-4aa0-bd53-55cef71c2115.png)
> 
> **Figma**
> 
> ![Screenshot from 2023-02-17 07-41-54](https://user-images.githubusercontent.com/105117832/219657182-0c7ca2e0-0863-4622-a0d8-349fce48c975.png)
> 
> ![Screenshot from 2023-02-17 07-42-06](https://user-images.githubusercontent.com/105117832/219657193-88c02f51-12f5-4396-bef6-400a10f9a446.png)

I didn't see the difference, I design similar with figma.

Reply:
@WasyihunS yes it is almost similar but as you can see in the nav bar image, the items inside of it are in the middle instead be taking all the width. And in the main page, the difference is the background